### DISCUSSION TITLE
imsai8080 ssid found but will not load 194.168.4.1

### DISCUSSION THREAD

**Discussion by jrd25** [thehighnibble/thehighnibble.github.io#179] [IMSAI8080esp replica]
I had this machine woeking perfectly till I decided to install the new S-132 board.  I relize now that this was a mistake because I am not savy enough about all this to make it work.  I decided to go back to the windows pc and GUI desktop etc.  I re-edited the conf file to its original state.  Now when I fire up the windows pc, I can get out to the internet using my home network.  I can also see the imsai8080 ssid as available network.  When I try and and connect to it, I get a box that says "connect automatically, and a button that says connect.  I am not asked to type a password.  I apparently get connected because I get a box that says "Connected, no internet". When I type imsai8080 or 194.168.4.1 in the address box, and get a message saying not found or something similar.  If anyone has idea of what the problem might be I would sure appreciate the help.  I appears that my windows machine recognizes the presense of the imsai8080 device, but won't connect.

Thanks in Advance for any help with this.

JRD25

**Comment by pfcushing** [comment]
it may have been a typo on your part, but the ip address for the imsai replica in AP mode is 192.168.4.1 not 194.168.4.1

Paul

**Reply by jrd25** [reply]
Thank you, Thank you for the info.  I will try that first thing tomorrow.

**Comment by jrd25** [comment]
I am happy to report that the problem is solved.  Thanks so much pfcushing.
